python-avd/pyavd/_eos_designs/structured_config/flows/__init__.py

```diff
@@ -111,11 +111,19 @@ def _set_hardware_flow_tracking(self) -> None:
         # Validate and configure trackers
         for tracker_name in natural_sort(trackers):
             config = self._get_tracker_input_config(tracker_name)
-            self.structured_config.flow_tracking.hardware.trackers.append_new(
+            flow_tracking_hardware_trackers = self.structured_config.flow_tracking.hardware.trackers.append_new(
```

Comment:
This will return only one tracker object
```suggestion
            flow_tracking_hardware_tracker = self.structured_config.flow_tracking.hardware.trackers.append_new(
```

Reply:
Fixed!